src/main/java/com/faforever/client/leaderboard/LeaderboardServiceImpl.java

```diff
@@ -27,6 +27,14 @@ public LeaderboardServiceImpl(FafService fafService) {
   }
 
   @Override
+  public int roundRatingToLowerHundred(double rating) {
+    if(rating < 0 ){
+      rating -= 100;
+    }
+    return (int)rating / 100 * 100;
```

Comment:
Please make sure that you have "Reformat code` enabled in your commit dialog (as well as "Rearrange code" and "Optimize imports"), and apply the formatter retroactively to all files changed in this PR.

And the method shouldn't be opened more than required;
```
@VisibleForTesting
int roundRatingToLowerHundred(double rating)
```

So no need for an interface method and mock implementation.

Reply:
I just installed the Save-Actions Plugin, so that should resolve any future formatting issues (I'm just used to formatting to apply on strg + s ^^)

The roundRatingToLowerHundred-Method is also used in the Ladder1v1Controller to round the player rating to find the matching bar to put the player.

Maybe the whole thing should be moved to the RatingUtil class instead? There is a getRoundedRating() - Method but that's behaving a bit differently: return (rating + 50) / 100 * 100;

Reply:
Oh now I see it, I missed that :) Yeah I guess it should go into the utility class. Maybe call it `getRoundedRatingBy100`, or allow to pass a rounding parameter (50, 100). And while we're at it, we should fix the existing `getRoundedRating` as well.

Reply:
Is there a reason to have these two different kinds of roundings? If not, going with the (fixed) one from the RatingUtil class might make things a lot clearer.